app/build.gradle.kts

```diff
@@ -85,6 +85,12 @@ dependencies {
 	implementation("com.github.bumptech.glide:glide:3.7.0")
 	implementation("com.flaviofaria:kenburnsview:1.0.6")
 
+	// HTTP utility
+	// NOTE: This is used by Picasso through reflection and can cause weird caching issues if removed!
+	val okhttpVersion = "2.7.0"
```

Comment:
Assuming we use 2.x because the libraries _might_ not work with newer versions. The latest 2.x version is [2.7.5](https://square.github.io/okhttp/changelog_2x/) so this needs to be bumped.

Reply:
I just reverted the original removal. I will get this updated later today.

Reply:
Version has been bumped.